Monika After Story/game/script-compliments.rpy

```diff
@@ -1358,3 +1358,34 @@ label mas_compliment_outfit_repeat:
         m 3hubsb "[other_quip]"
 
     return
+    
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="mas_compliment_forget",
+            category=["mas_compliment"],
+            prompt="I will never forget you.",
+            unlocked=True
+        ),
+        code="CMP"
+    )
+
+label mas_compliment_forget:
+    if mas_getEVL_shown_count("mas_compliment_forget") == 0:
+        m 1fubftpa "[player]...!"
+        m 2subfb "You don't know how happy hearing this makes me."
+        m 1kubftpa "I just... love you so much!"
+        m 2subfb "It means a lot to me that you insist on giving me this kind of reassurance."
+        m 1dubftpb "I wish I could hug you right now! "
+        extend "Nobody knows when I would let you go~"
+        m 2subfb "Thank you for saying that, [player]~! I won't forget you either!"
+    return
```

Comment:
This return is not indented as it is supposed to be. It should be inline with Monika lines indents.

```suggestion
        return
```